tst/com/amazon/corretto/crypto/provider/test/AesTest.java

```diff
@@ -538,7 +637,7 @@ public void test_bufferOverflows() throws Throwable {
         sneakyInvoke(spi, "engineInit", Cipher.ENCRYPT_MODE, key,
                 new GCMParameterSpec(128, randomIV()), rnd);
         assertThrows(ShortBufferException.class,
-                () -> sneakyInvoke(spi, "engineUpdate", new byte[16], 0, 16, new byte[32], 16));
```

Comment:
Same comment, why 16 -> 17?

Reply:
Same answer: This is actually testing for the buggy behavior. Since the buggy behavior was never actually required, we are removing the test.